JesusFilm/core
# feat: videojs youtube

# Description

Enable the support of youtube videos in journeys and journeys-admin. We are doing this via the plugin videojs-youtube

[add videoURL field to background video ](https://3.basecamp.com/3105655/buckets/29264588/todos/5317254351)
[add videoURL to video fullscreen and video in card](https://3.basecamp.com/3105655/buckets/29264588/todos/5317256078)
[Youtube to run the same video events](https://3.basecamp.com/3105655/buckets/29264588/todos/5317256974)

# How should this PR be QA Tested?

If not merged with Admin changes. Go to Arango. And in Arango, update video block's videoId and title (title optional for background but needed for video - needed for events)

- [ ] Youtube videos should work as a background video
- [ ] Youtube videos should work when in video fullscreen mode
- [ ] Youtube videos should work in video component

**Functionalities to work as expected** - if not merged with Admin, please update data acordingly in Arango
- [ ] Video Events
- [ ] StartAt
- [ ] EndAt
- [ ] NextBlock (jump to next step)
- [ ] Fullscreen
- [ ] Muted
- [ ] Autoplay

# Checklist
